lib/cadet_web/controllers/user_controller.ex

```diff
@@ -22,20 +24,75 @@ defmodule CadetWeb.UserController do
       grade: grade,
       max_grade: max_grade,
       story: story,
-      xp: xp
+      xp: xp,
+      game_states: game_states
     )
   end
 
   swagger_path :index do
     get("/user")
-
     summary("Get the name and role of a user")
-
     security([%{JWT: []}])
+    produces("application/json")
+    response(200, "OK", Schema.ref(:UserInfo))
+    response(401, "Unauthorised")
+  end
 
+  def update_game_states(conn, %{"gameStates" => new_game_states}) do
+    user = conn.assigns[:current_user]
+
+    case Cadet.GameStates.update(user, new_game_states) do
```

Comment:
`Cadet.GameStates` is already imported so you can just use `update` here. Alternatively you can use `alias Cadet.GameStates` and type `GameStates.update`

Same thing for `Cadet.GameStates.clear` below at line 76